Added graph and minor fixes (#4)
thesis20/thesis_paper: sections/csgcn/csgcn-chapter.tex

\\
There are a number of graph convolution layers that update the representation of each central node with information about their neighbor nodes.
The representation at layer $l+1$ is calculated by taking the normalized sum of the representations of neighbor nodes at layer $l$. 
As mentioned, neighbor nodes for user are item nodes and user side-information nodes, and for items the neighbors are user nodes and item side-information nodes, which can be seen on \Cref{fig:quadripartite-graph}.

> For each additional layer, information is passed from nodes that are further away from a central node, which can be seen on \Cref{fig:aggregation-for-layers}

Reviewer: ```suggestion
For each additional layer, information is passed from nodes that are further away from a central node, illustrated on \Cref{fig:aggregation-for-layers}
```
